scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 547,
-  MAX_SIZE: 585,
+  MIN_SIZE: 639,
```

Comment:
Continuing from https://github.com/bbc/simorgh/pull/4184/files#r334420000

Reply:
Attempting the conditionally loading of `@bbc/psammead-rich-text-transforms` in the POC PR - https://github.com/bbc/simorgh/pull/4240